voorhoede/head-start
# feature: upgrade to Astro v5 & Node.js v22

# Changes

- Upgrade to Astro v5 (stable) and Node.js v22 (latest LTS version).
- Upgrade related packages (adapters, linter, etc).

# Associated issue

Resolves #188 

# How to test

1. Open preview link
2. Verify all pages still work
3. Verify preview mode still works

# Checklist
